cmd/integration_test/build/buildspecs/tinkerbell-test-eks-a-cli.yml

```diff
@@ -30,6 +30,11 @@ env:
     T_REGISTRY_MIRROR_USERNAME_TINKERBELL: "harbor-registry-data:tinkerbell_username"
     T_REGISTRY_MIRROR_PASSWORD_TINKERBELL: "harbor-registry-data:tinkerbell_password"
     T_REGISTRY_MIRROR_CA_CERT_TINKERBELL: "harbor-registry-data:tinkerbell_caCert"
+    T_PRIVATE_REGISTRY_MIRROR_ENDPOINT_TINKERBELL: "harbor-registry-data:authenticated_tinkerbell_endpoint"
+    T_PRIVATE_REGISTRY_MIRROR_PORT_TINKERBELL: "private-harbor-registry-data:authenticated_tinkerbell_port"
+    T_PRIVATE_REGISTRY_MIRROR_USERNAME_TINKERBELL: "private-harbor-registry-data:authenticated_tinkerbell_username"
+    T_PRIVATE_REGISTRY_MIRROR_PASSWORD_TINKERBELL: "private-harbor-registry-data:authenticated_tinkerbell_password"
+    T_PRIVATE_REGISTRY_MIRROR_CA_CERT_TINKERBELL: "private-harbor-registry-data:authenticated_tinkerbell_caCert"
```

Comment:
should these come from `harbor-registry-data` instead of `private-harbor-registry-data`?